src/styles/index.scss

```diff
@@ -1,3 +1,66 @@
-body {
+@import './utils/variables';
+@import './utils/mixin';
+
+* {
   margin: 0;
+  box-sizing: border-box;
+}
+
+.page {
+  @include center-flex;
+
+  height: 100vh;
+}
+
+.calendar {
+  width: 7 * $block-size + 6 * $gap + 2 * $padding;
+  gap: $gap;
+  padding: $padding;
+  display: flex;
+  flex-wrap: wrap;
+
+  @each $name, $day in $days {
+    &--start-day-#{$name} {
+      .calendar__day:first-child {
+        margin-left: ($block-size + $gap) * $day;
+      }
+    }
+  }
+
+  @for $i from 28 through 31 {
+    &--month-length-#{$i} {
+      .calendar__day:nth-child(n + #{$i + 1}) {
+        display: none;
+      }
+    }
+  }
+
+  &__day {
+    @include center-flex;
+
+    background-color: $main-color;
+    width: $block-size;
+    height: $block-size;
+    border: 1px black solid;
+
+    &:first-child {
+      margin-left: ($block-size + $gap) * 6;
+    }
+
+    @for $i from 1 through 31 {
+      &:nth-child(#{$i})::before {
+        content: '#{$i}';
+        font-family: Arial, Helvetica, sans-serif;
+        font-size: 30px;
+      }
+    }
+
+    transition: all 0.3s;
```

Comment:
Not to use transitions for everything. 
It is better to specify which properties to apply the transition to.
![image](https://github.com/mate-academy/layout_calendar/assets/54860641/c099f68d-ac3b-4c69-896e-c20037c8b8e7)